api/sip/client.py

```diff
@@ -405,10 +367,54 @@ def login_response_parser(self, message):
             fixed.login_ok
         )
 
+    def end_session_message(
+            self, patron_identifier, patron_password="", institution_id="",
+            terminal_password="",
+    ):
+        """
+        This message will be sent when a patron has completed all of their
+        transactions. The ACS may, upon receipt of this command, close any
+        open files or deallocate data structures pertaining to that patron.
+        The ACS should respond with an End Session Response message.
+
+        Format of message to send to ILS:
+        35<transaction date><institution id><patron identifier>
+        <terminal password><patron password>
+        transaction date: 18-char, YYYYMMDDZZZZHHMMSS, required
+        institution id: AO, variable length, required
+        patron identifier: AA, variable length, required
+        terminal password: AC, variable length, optional
+        patron password: AD, variable length, optional
+        """
+        code = "35"
+        timestamp = self.now()
+
+        message = (code + timestamp +
+                   "AO" + institution_id + self.separator +
+                   "AA" + patron_identifier + self.separator +
+                   "AC" + terminal_password
+        )
+        if patron_password:
+            message += self.separator + "AD" + patron_password
+        return message
+
+    def end_session_response_parser(self, message):
+        """Parse the response from a end session message."""
+        return self.parse_response(
+            message,
+            36,
+            fixed.end_session,
+            fixed.transaction_date,
+            named.institution_id.required,
+            named.patron_identifier.required,
+            named.screen_message,
+            named.print_line
+        )
+
     def patron_information_request(
             self, patron_identifier, patron_password="", institution_id="",
             terminal_password="",
-            language=None, summary=None, start_item="", end_item=""
+            language=None, summary=None
```

Comment:
What's the effect of this change?

Reply:
The parameters weren't used in the function.

Reply:
:ok: